app/controllers/promo_registrations_controller.rb

```diff
@@ -14,20 +14,11 @@ def create
     @publisher = current_publisher
     @publisher.promo_enabled_2018q1 = true
     @publisher.save!
-    @publisher_has_verified_channel = @publisher.has_verified_channel?
-
-    if @publisher_has_verified_channel
-      if @publisher.channels.where(verified: true).count > 5
-        Promo::RegisterPublisherForPromoJob.perform_later(publisher: @publisher)
-        redirect_to home_publishers_path, notice: t("promo.activated.please_wait")
-      else
-        Promo::PublisherChannelsRegistrar.new(publisher: @publisher).perform
-        @promo_enabled_channels = @publisher.channels.joins(:promo_registration)
-        PromoMailer.promo_activated_2018q1_verified(@publisher, @promo_enabled_channels).deliver
-      end
-    else
-      PromoMailer.promo_activated_2018q1_unverified(@publisher).deliver
+    current_publisher.channels.left_joins(:promo_registration).where(promo_registrations: {id: nil}).find_each do |channel|
```

Comment:
It wasn't exactly obvious what this query was doing to me. Since the `AssignPromoToChannelService` job is checking to see if the channel is verified and there isn't a promo_registration already on the channel can we simplify this and call `register_channel_for_promo` on all channels so this code is easier to understand?